### PR TITLE
Fix scheduling of docker test for SLE12-SP3 and SLE12-SP4

### PR DESCRIPTION
**Pull request by pdostal**
Hello,
there's a problem with scheduling. I didn't see it in #6633 so I'd like to fix it now.

- Related ticket: None
- Needles: None
- Verification run: [SLE12sp3](
http://pdostal-server.suse.cz/tests/1099) [SLE12sp4](
http://pdostal-server.suse.cz/tests/1098) [SLE15](
http://pdostal-server.suse.cz/tests/1100)

### CODE REVIEW

**Comment by Vogtinator** [comment]
It looks like @sysrich applied neither your correction nor my [suggested change](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6633#discussion_r251138376).

This PR is incomplete as well though, it does not fix `stable_names`.

**Comment by pdostal** [comment]
@Vogtinator can you please copy your suggestion here? I cannot see the change properly.

File: lib/suse_container_urls.pm [hunk 1]
```diff
@@ -33,12 +33,12 @@ sub get_suse_container_urls {
...
-    if (is_sle("=12-sp3") && is_sle('=12-SP4')) {
+    if (is_sle(">=12-sp3") && is_sle('<=12-SP4')) {
         my $lowerversion  = lc $version;
-        my $nodashversion = $version =~ s/-SP/sp/r;
+        my $nodashversion = $version =~ s/-sp/sp/ir;
         # No aarch64 image
         if (!check_var('ARCH', 'aarch64')) {
-            push @image_names,  "registry.suse.de/suse/sle-${lowerversion}/docker/update/cr/images/suse/sle-${nodashversion}";
+            push @image_names,  "registry.suse.de/suse/sle-${lowerversion}/docker/update/cr/images/suse/sles${nodashversion}";
             push @stable_names, "registry.suse.com/suse/${nodashversion}";
```
Comment (Vogtinator) [inline lib/suse_container_urls.pm]:
```suggestion
            push @stable_names, "registry.suse.com/suse/sles${nodashversion}";
```

**Comment by Vogtinator** [comment]
> @Vogtinator can you please copy your suggestion here? I cannot see the change properly.

Done

**Comment by Vogtinator** [comment]
> @Vogtinator is that OK now?

LGTM

**Comment by sysrich** [comment]
Thanks to both of you :)

**Pull request merged**